### ISSUE TITLE
Add ability to continue training process after aborting

### ISSUE THREAD

**Issue by drozdvadym** [NVIDIA/DIGITS#24]
Now, when server stopped or there was another problem, I have only one method to continue training:
* Creating new task based on previous

Perhaps, better to add this ability to the aborted task?

**Comment by lukeyeager** [comment]
I totally agree, this should be a feature in the future. For now, you can do the same thing by retraining. When creating a new model, select the previous model that got aborted under the `Previous Networks` tab, and start with the weights at `Epoch #30`, or whatever the last snapshot was.

Technically, this is a little different because you lose the momentum and other solver state variables, but this will essentially get you what you want until this feature gets implemented.

**Comment by jmozah** [comment]
i bumped in to the same problem today and was about to open a ticket ... then i spotted this... Thanks for the tip to restart using the "Previous Networks" :-)

**Comment by Tiaguituh05** [comment]
Hey @lukeyeager I was tranning a model with 30 epochs, it was on the epoch 12 when the pc shut down for some reason (it still had 2 days till the training was finish). So I created a new model and in the previous networks tab I chose the epoch 12 on the model it was training before. So it started a new training, but it says it will take 6 days while the other job only had 2 days remaining, am I doing something wrong?

**Comment by Pfaeff** [comment]
Are there technical reasons for why this is still not implemented?

**Comment by tlf30** [comment]
We are currently doing very long training batches with DIGITS, and have run into the scenario where we need to do server maintenance, but do not want to loose the progress we have made. We have also run into unexpected shutdowns which have killed our training. Is it possible to get pausing and resuming implemented, I am surprised that it is not currently. Even if it only starts back at the last epoch, it would be a large improvement.
